mava/components/tf/networks/continuous.py

```diff
@@ -0,0 +1,152 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Networks used in continous settings.
+
+Some networks adapted from
+https://github.com/deepmind/acme/blob/master/acme/tf/networks/continuous.py.
+"""
+from typing import Any, Optional, Sequence
+
+import sonnet as snt
+import tensorflow as tf
+from acme import types
+from acme.tf import utils as tf2_utils
+from acme.tf.networks.continuous import ResidualLayernormWrapper
+
+
+def get_initialization(seed: Optional[int] = None) -> Any:
```

Comment:
What makes all these new components different from Acme? Is it to be able to use a seed?

Reply:
Yep. For most of this it is the seed and the networks are more configurable, i.e. layernorm and activations are configurable.